st2actions/st2actions/cmd/scheduler.py

```diff
@@ -109,7 +109,7 @@ def _run_scheduler():
             handler.shutdown()
             entrypoint.shutdown()
         except:
-            pass
+            LOG.exception('Unable to shutdown scheduler.')
```

Comment:
This error was / is not fatal before so we shouldn't use log.exception which results in error log level. Should probably just use info log level + add original error message (or maybe even debug + exc_info=True).